src/containers/create-post/create-post.component.jsx

```diff
@@ -8,18 +8,21 @@ const CreatePost = (props) => {
   const [postText, setPostText] = useState("");
   const [option1, setOption1] = useState("");
   const [option2, setOption2] = useState("");
-  const userID = useSelector((state) => state.users.user.userId);
+  const user = useSelector((state) => state.users.user);
+  const userID = user.userId;
+  const username = user.username;
+  const avatarLink = user.avatar_link;
 
   const addPostToDB = () => {
-    addPost(postText, option1, option2, userID);
+    addPost(postText, option1, option2, userID, username, avatarLink);
```

Comment:
I know this is old code but can we add the object params here too?

Reply:
Yup, figured you'd comment on that

Reply:
What is the benefit of that btw? i dont really know

Reply:
DID tho